opendcs-rest-api/src/main/java/org/opendcs/odcsapi/res/OpenDcsResource.java

```diff
@@ -81,7 +84,19 @@ final OpenDcsDatabase createDb()
 			{
 				throw new IllegalStateException("Error connecting to the database via JNDI", ex);
 			}
-//			throw new IllegalStateException("Error connecting to the database via JNDI", e);
+			//			throw new IllegalStateException("Error connecting to the database via JNDI", e);
```

Comment:
If its handled, just delete this commented out throw call.

Reply:
Removed